src/main/scala/com/typesafe/sbt/packager/archetypes/jlink/JlinkPlugin.scala

```diff
@@ -146,7 +146,7 @@ object JlinkPlugin extends AutoPlugin {
 
       IO.delete(outDir)
 
-      runForOutput(run("jlink", jlinkOptions.value), log)
+      run("jlink", jlinkOptions.value)
```

Comment:
This is the sbt `ForkRun` method you mentioned in the ticket description, right?

Reply:
No, that is a convenience alias for `runJavaTool`. The `ForkRun` thing required too much `Compat` noise to be worthwhile. So I've reverted the MR to use `ProcessBuilder`s.

 That idea can be revisited once we drop SBT 0.13.